### DISCUSSION TITLE
How to set the locale for VuetifyDateAdapter?

### DISCUSSION THREAD

**Discussion by 1Luc1** [vuetifyjs/vuetify#17293] [Q&A]
Ask shown within the docs, it is possible to set the locale for an extern DateAdapter, but I couldn't figure it out, how to set it for the VuetifyDateAdapter. https://vuetifyjs.com/en/features/dates/#usage .

**Comment by KaelWD** [comment]
Looks like it uses the locale from https://vuetifyjs.com/en/features/internationalization/ and you can't actually pass an instance like https://vuetifyjs.com/en/features/dates/#adapter shows, only a constructor


```diff
  import { createVuetify } from 'vuetify/labs'
+ import { sv } from 'vuetify/locale'
  import LuxonAdapter from "@date-io/luxon"

- const luxon = new LuxonAdapter({ locale: "sv" });

  export default createVuetify({
    date: {
-     adapter: luxon,
+     adapter: LuxonAdapter,
    },
+   locale: {
+     locale: 'sv',
+     messages: { sv },
+   },
  })
```

**Reply by 1Luc1** [reply]
Thank you. I couldn't test it yet, but will it work with the `createVueI18nAdapter` ?

```js
const vuetify = createVuetify({
  ....
  locale: {
    adapter: createVueI18nAdapter({ i18n, useI18n })
  }
 ...
})
```
From the example [here](https://vuetifyjs.com/en/features/internationalization/#vue-i18n).
